test-suite/exonum-py-tests/suite/temp_dir.py

```diff
@@ -0,0 +1,25 @@
+"""TempDir module."""
+import tempfile
+import shutil
+
+
+class TempDir:
+    """Class creating and managing the temporary directorions."""
+
+    def __init__(self, directory: str):
+        self.temp_dir = directory
+
+    @classmethod
+    def create(cls) -> "TempDir":
+        """Initializes a TempDir object."""
+        temp_dir = tempfile.mkdtemp(prefix="exonum_test_suite_")
+
+        return cls(temp_dir)
+
+    def path(self) -> str:
+        """Returns the path of the temporary dir."""
+        return self.temp_dir
+
+    def remove(self) -> None:
```

Comment:
Just out of curiosity. Do we really need the explicit method `remove` ? Isn't enough a `__exit__` method ?

Reply:
With that method user will be able to use `TempDir` without context manager, if he will ever want to, just like you can open a file either with
```python
with open(filename) as some_file:
    do_magic(some_file)
```

or manually:
```python
some_file = open(filename)
do_magic(some_file)
some_file.close()
```

It is usually good idea to support both workflows, since we aren't always able to use context managers (e.g. if object is a class field).